.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "haskell/adl"]
 	path = haskell/adl
-	url = git@github.com:timbod7/adl.git
+	url = https://github.com/timbod7/adl.git
```

Comment:
Why is this necessary? Doesn't this mean I'll need to start using http checkouts when working locally?

Reply:
This was to workaround the github action trying to pull from ssh.
Thing to do would be to revert this and try again.